src/tmx/TmxUtils/src/simulation/SimulationUtils.cpp

```diff
@@ -0,0 +1,30 @@
+#include <simulation/SimulationEnvVar.h>
+
+namespace tmx::utils::sim{
+    bool is_simulation_mode() {
```

Comment:
Also please pull develop and resolve merge conflicts. This file has been renamed to SimulationEnvUtils so merging this PR as it currently stands will include an unnecessary file.